openc3/lib/openc3/interfaces/http_server_interface.rb

```diff
@@ -31,6 +31,10 @@ def initialize(port = 80)
       @request_queue = Queue.new
     end
 
+    def connection_string
+      return "listening on #{@port}"
```

Comment:
Needs hostname too.

Reply:
Hostname isn't part of the connection parameters. It's just: `@server = WEBrick::HTTPServer.new :Port => @port`

Reply:
It should be though.  Should work the same as tcpip_server_interface.